### ISSUE TITLE
RxReachability with Carthage

### ISSUE THREAD

**Issue by alexth** [RxSwiftCommunity/RxReachability#10]
Hey @bmoliveira .
Thanks for RxReachability!
I have a question about using RxReachability with Carthage. As I see, original Reachability class from SwiftReachability is declared with `public` visibility. As I am using `RxReachability` as a Carthage module, I assume that it should be `open` instead. As a result, I'm able to import `RxReachability` in my swift file but Reachability.swift itself is not accessible. Would be glad to receive an advice on the topic.

Thanks in advance, 
Alex

**Comment by JoeMatt** [comment]
I had to include both manually,

```
github "RxSwiftCommunity/RxReachability"
github "ashleymills/Reachability.swift"
```

I'll see about making a fork and PR since I find this irritating and RxReachability doesn't work with Carthage's caching mechanism for some reason. Each boot strap rebuilds `RxReachability` even with `--cache-builds` on.

**Comment by bmoliveira** [comment]
I'm not sure how to be able to export to other libs the base lib of Reachability through Carthage, do you have any tips? 

Because most of my libs I get them and distribute them primarily through CocoaPods.

**Comment by tiagoribeiro4** [comment]
@bmoliveira I encountered this problem now that I'm trying to integrate your framework. I believe you just have to add a Cartfile with: `github "ashleymills/Reachability.swift"`

**Comment by JoeMatt** [comment]
closed by #46
